Autopilot/PathManager/Inc/CommWithAttitudeManager.hpp

```diff
@@ -0,0 +1,28 @@
+#ifndef GET_FROM_PATH_MANAGER_HPP
+#define GET_FROM_PATH_MANAGER_HPP
```

Comment:
This guard belongs to another file. Rule for these guards is either to just use #pragma once, which does the same thing or name the define YOUR_FILE_NAME_HPP. Since we guarantee no 2 files should have the same name, we guarantee no 2 guards have the same name either.